Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
```

I'd put a LTS version at least there to make the build consistent
```suggestion
FROM ubuntu:22.04
```